### ISSUE TITLE
gas push not found

### ISSUE THREAD

**Issue by Identity-labs** [MaartenDesnouck/google-apps-script#74]
Hi, 

I can't push au datastudio project with gas.
The project was made directly in google drive and clone after with gas.

```
identity-labs:gasMyposeo $ gas deploy
Pushing 'myposeo' to Google Drive... [✘]
The code you are trying to push appears to have 1 or more syntax error(s). Push was reverted [✘]
----------------------------------------------------------------------------------------

appsscript.js:2
  "timeZone": "Europe/Paris",
            ^
ParseError: Unexpected token
```

The content of appscript.js

```
{
  "timeZone": "Europe/Paris",
  "dataStudio": {
    "name": "MYPOSEO Campaign",
    "company": "myposeo",
    "logoUrl": "http://www.myposeo.com/wp-content/uploads/2015/10/myposeo_logo_2016.png",
    "addonUrl": "https://developers.google.com/datastudio/connector/getstarted",
    "supportUrl": "https://developers.google.com/datastudio/connector/faq",
    "description": "This connector uses the myposeo API to retrieve metadata for all campaigns."
  }
}
```

Thanks for your help

**Comment by MaartenDesnouck** [comment]
When I try to save this piece of code you posted in the online editor Google Drive tells me I've made a mistake. Is this the full content of appscript.js? And is this the only file you are trying to push?

**Comment by wulftone** [comment]
See #78 for more details, same problem.  This is supposed to be a json file

**Comment by MaartenDesnouck** [comment]
@Identity-labs Should be fixed in 2.4.0, let me know if it works.
("npm i -g google-apps-script" to get the latest version)
